test/lib/BedrockTester.cpp

```diff
@@ -583,4 +583,36 @@ int BedrockTester::waitForPort(int port) {
     return 1;
 }
 
+STable BedrockTester::getStatus(bool control) {
+    // FYI: sometimes the status command doesn't return with every key/value expected
+    SData request("Status");
```

Comment:
i don't think you need this variable at all:

`return SParseJSONObject(executeWaitVerifyContent(SData("Status"), "200", control));`